src/Form/MetadataDisplayForm.php

```diff
@@ -201,10 +201,29 @@ public static function ajaxPreview($form, FormStateInterface $form_state) {
         }
       }
     }
+
+    $output = [];
+    $output['json'] = [
+      '#type' => 'details',
+      '#title' => t('JSON Data'),
+      '#open' => false,
+    ];
+    $output['json']['data'] = [
+      '#type' => 'codemirror',
+      '#rows' => 60,
+      '#value' => json_encode($context['data'], JSON_PRETTY_PRINT),
```

Comment:
This is actually quite cool. The fact that you are showing the whole $context and not only the SBF JSON. Just noticed it and makes me =) Thanks!